### ISSUE TITLE
module libxml2... not found

### ISSUE THREAD

**Issue by kladitt** [itstool/itstool#46]
I try to install itstool-2.0.7and get this error:

checking for /usr/bin/python script directory... ${prefix}/lib/python3.10/site-packages
checking for /usr/bin/python extension module directory... ${exec_prefix}/lib/python3.10/site-packages
checking for python module libxml2... not found
configure: error: Python module libxml2 is needed to run this package

In /usr//lib/python3.10/site-packages I have:
/usr//lib/python3.10/site-packages/lxml
/usr//lib/python3.10/site-packages/lxml-4.8.0-py3.10-linux-x86_64.egg
/usr//lib/python3.10/site-packages/lxml-4.8.0.dist-info

In /usr/lib python and python3 are symbolic links to python3.10

Where can i find the sources to install this module libxml2? 
pip3 does not know it.

libxml2-2.9.13 installed :
/usr/local/lib/python3.10/site-packages/libxml2mod.la
/usr/local/lib/python3.10/site-packages/drv_libxml2.py
/usr/local/lib/python3.10/site-packages/libxml2.py
/usr/local/lib/python3.10/site-packages/libxml2mod.so
but no module libxml2.

I am confused and any help is welcome.

**Comment by zhw2101024** [comment]
It's because libxml2 has disabled the python binding by default, and planned to remove it on the next release.
Some changed on the itstool side is needed now.

**Comment by thesamesam** [comment]
That's not actually the case for the original report, which was from 2022 and also shows the module being installed, but indeed true in 2025.

**Comment by kladitt** [comment]
Indeed! Changing libxml2 by lxml does not help because the modules are different in functionality.
